### PR TITLE
Support agent-group config cli

### PR DESCRIPTION
**Pull request by SongZhen0704**
**Phenomenon and reproduction steps**

**Root cause and solution**

- metaflow-ctl agent-group create -f xxx.yaml
- metaflow-ctl agent-group update -f xxx.yaml
- metaflow-ctl agent-group delete g-xxxx

**Impactions**

**Test method**

./metaflow-ctl agent-group-config list
./metaflow-ctl agent-group-config list woFYi5035l
./metaflow-ctl agent-group-config list -o yaml
./metaflow-ctl agent-group-config list woFYi5035l -o yaml
./metaflow-ctl agent-group-config create -f vtap_group_create.yaml
./metaflow-ctl agent-group-config update woFYi5035l -f vtap_group_create.yaml

**Affected branch(es)**

* main

**Checklist**

- [ ] Dependencies update required
- [ ] Common bug (similar problem in other repo)

### CODE REVIEW

File: cli/ctl/agent_group_config.go [hunk 1]
```diff
@@ -14,28 +15,187 @@ func RegisterAgentGroupConfigCommand() *cobra.Command {
...
-			fmt.Printf("please run with 'example | list | create | delete'.\n")
+			fmt.Printf("please run with 'example | list | create | update | delete'.\n")
+		},
+	}
+
+	var listOutput string
+	list := &cobra.Command{
+		Use:     "list [agent-group ID]",
+		Short:   "list config",
+		Example: "metaflow-ctl agent-group-config list deepflow-config",
```
Comment (sharang) [inline cli/ctl/agent_group_config.go:26]: 这个例子最后的一个参数是什么含义，是否要改为metaflow-config

File: cli/ctl/agent_group_config.go [hunk 2]
```diff
@@ -14,28 +15,187 @@ func RegisterAgentGroupConfigCommand() *cobra.Command {
...
+			listAgentGroupConfig(cmd, args, listOutput)
+		},
+	}
+	list.Flags().StringVarP(&listOutput, "output", "o", "", "output format")
+
+	var createFilename string
+	create := &cobra.Command{
+		Use:     "create -f <filename>",
+		Short:   "create config",
+		Example: "metaflow-ctl agent-group-config create -f deepflow-config.yaml",
```
Comment (sharang) [inline cli/ctl/agent_group_config.go:37]: 同上

File: cli/ctl/agent_group_config.go (hunk 1 above)
Comment (SongZhen0704) [reply cli/ctl/agent_group_config.go:26]: 这里应该是agent-group ID，我修改一下

File: cli/ctl/agent_group_config.go (hunk 2 above)
Comment (SongZhen0704) [reply cli/ctl/agent_group_config.go:37]: 这里是yaml文件的命令，我修改为metaflow-config

**Review by sharang — APPROVED** [review]

**Pull request merged**